### ISSUE TITLE
Could you please share problem statements from codeforces.com?

### ISSUE THREAD

**Issue by anks12297** [Sumith1896/spoc#2]
Hi, thanks for sharing your great work!

I have read your paper and downloaded spoc dataset. I am not able to find the actual problem statement collected from codeforces.com in dataset. Could you please share real problem statements for all 677 problems. This would be great help to me.

Thank you.

**Comment by sumith1896** [comment]
Hi @anks12297, thank you for your interest in the SPoC dataset! Writing this on behalf of Ice as well: 

We only scraped the submitted code and the test cases, and as such we did not have the problem statements. If you want to scrape the problem statements yourself, the dataset contains the problem IDs which can be used to look up the corresponding problem pages on CodeForces.

I have pushed the scraping script to master (under /scripts). The usage is `python scrape.py START_PAGE END_PAGE`. We ran this in a bash for loop with START_PAGE = END_PAGE running from 1 to 49 in parallel. The script might break if Codeforce changed the page layout.

Let us know if you have further questions.
